src/main/java/com/dajudge/kindcontainer/KindContainer.java

```diff
@@ -243,6 +247,7 @@ private void untaintNode() {
         });
     }
 
+    @SuppressWarnings("SameParameterValue")
```

Comment:
I think this ended up here by accident?

Reply:
As I explained in the PR description, I looked into adding the same functionality for the `KindContainer`. That's why I did a cleanup of that class first. Dropped the commit in this PR.